main.css

```diff
@@ -98,4 +98,47 @@ body{
             max-height: 50%;
             height: 50vh!important;
     }
-}
\ No newline at end of file
+}
+
+/* AUTHORS - CSS code*/
+
+.section-authors {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    height: 87%;
+    width: 100%;
+    background-color: brown;
+    text-align: center;
+}
+
+.section-authors-title {
+    font-family: ‘LABRIT’;
```

Comment:
use font-family if the  first one will not load ->  e.g. `font-family: ‘LABRIT’, sans-serif;`